docs/configuration/response_body_processor.md

```diff
@@ -6,9 +6,20 @@ The processing is done to extract relevant information from the responses and st
 
 Iguana supports multiple response body processors that are defined by the content type of the response body they process.
 
-Currently only the `application/sparql-results+json` content type is supported, 
-and it only uses the `SaxSparqlJsonResultCountingParser` language processor 
-to extract simple information from the responses.
+The following content types are supported:
+- `application/sparql-results+json`
+- `application/sparql-results+xml`
+- `text/csv`
+- `text/tab-separated-values`
```

Comment:
Please create another issue to also support graphs in the form of Turtle, RDF/XML, RDF/JSON, N-Ttriples and Turtle coming form CONSTRUCT queries.